<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>210</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>370</width>
    <height>130</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>NXController</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="devicelist">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Refresh">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>18</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Disconnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_Serial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_Socket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP (sys-botbase)</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Address">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address 0x</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Data">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>145</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Datasize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of bytes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Read">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Write">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipaddress">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>192.168.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ramaddress">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="data">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="datasize">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Settings">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_UseMouse">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Use Mouse</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_MousePad">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>380</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Press F9 to Stop</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
